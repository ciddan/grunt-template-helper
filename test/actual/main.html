<div class="foo">
  <h1>Single</h1>
</div>