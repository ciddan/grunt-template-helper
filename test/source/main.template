<div class="<%= class %>">
  <h1><%= message %></h1>
</div>