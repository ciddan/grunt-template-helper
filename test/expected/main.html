<div class="foo">
  <h1>Hello</h1>
</div>